introduction.md

```diff
@@ -1,2 +1,7 @@
 This is an introduction
-My favourite Code School path is the Ruby path
\ No newline at end of file
+My favourite Code School path is the Ruby path
+Deadly skills:
+* Git
+* Ruby
+* CSS
+* Javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?